agot-bg-game-server/src/client/CombatInfoComponent.tsx

```diff
@@ -38,6 +39,11 @@ export default class CombatInfoComponent extends Component<CombatInfoComponentPr
     render(): ReactNode {
         return (
             <>
+                {this.props.showTitle && (
+                    <div style={{display: 'flex', justifyContent: 'center'}}>
+                        <h5>Battle for <strong>{this.defender.region.name}</strong></h5>
+                    </div>
+                )}
```

Comment:
If the title is only shown in the GameState Planel and not in the game log, why not adding it in the `CombatComponent`?

Reply:
Indeed. That would simplify it completely. I close this PR as the branch name does not fit anymore. Creating a new one just for the 3 lines seems a bit ridiculous to me so if you want you could add the title to the combat component by just copying the 3 lines of code from here when you do the highlighting thing?